app/src/main/java/com/github/polypoly/app/base/game/rules_and_lobby/GameLobby.kt

```diff
@@ -0,0 +1,97 @@
+package com.github.polypoly.app.base.game.rules_and_lobby
+
+import com.github.polypoly.app.base.game.Game
+import com.github.polypoly.app.base.user.User
+import kotlin.time.Duration.Companion.hours
+
+/**
+ * Represent a game lobby where [User]s can join and wait for the game to start,
+ * and where the [admin] can decide some [rules] about the [Game]
+ * @property admin The admin of the lobby who have created the [GameLobby]
+ * @property rules The rules of the future [Game]
+ * @property name The name of the [GameLobby]
+ * @property code The (secret) code of the [GameLobby]
+ * @property private If the [GameLobby] is private or not
+ */
+class GameLobby(
+    val admin: User = User(),
+    val rules: GameRules = GameRules(),
+    val name: String = "",
```

Comment:
Let's be sure that the default name respects what we need (not empty nor blank), so it could be something like "default"

Reply:
done